### DISCUSSION TITLE
Is this detectable?

### DISCUSSION THREAD

**Discussion by TheBloonsGuy** [IMXNOOBX/cs2-external-esp#54] [General]
I wanted to know if this cheat is detectable. Anyone knows?

**Comment by ifBars** [comment]
> I wanted to know if this cheat is detectable. Anyone knows?

Any cheat that is free and public is easily detectable, and any cheat can be detectable. I think what you are trying to ask is if it is currently detected. The answer to that, is probably not, but find out for yourself. It doesn't write memory just reads, so far I haven't heard of any bans while using it, but that could change any day.

**Reply by TheBloonsGuy** [reply]
Yeah i meant if it can be detected but probably not untill an update

**Reply by ifBars** [reply]
It can be detected yes, but currently VAC in CS2 kind sucks, so no it is not detected, **FOR NOW**
